docs/command/atlas-federatedAuthentication-federationSettings-describe.txt

```diff
@@ -0,0 +1,86 @@
+.. _atlas-federatedAuthentication-federationSettings-describe:
+
+=========================================================
+atlas federatedAuthentication federationSettings describe
+=========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Return the Federation Settings details for the specified organization.
+
+To use this command, you must authenticate with a user account or an API key with the Organization Owner role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas federatedAuthentication federationSettings describe [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for describe
+   * - --orgId
+     - string
+     - false
+     - Organization ID to use. This option overrides the settings in the configuration file or environment variable.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see the full output, use the -o json option.
```

Comment:
Where possible, literals and terminal command text should be in backticks. (Do we do this elsewhere in the these particular CLI docs?)

```suggestion
     - Output format. Valid values are ``json``, ``json-path``, ``go-template``, or ``go-template-file``. To see the full output, use the ``-o json`` option.
```

Reply:
This applies throughout this PR.

Reply:
Since output is a flag already in-place, let's add this change in another PR because it will impact all commands in the CLI. Here's a ticket tracking that https://jira.mongodb.org/browse/CLOUDP-245249